src/docfx/config/ops/OpsConfigAdapter.cs

```diff
@@ -76,8 +76,8 @@ private async Task<string> GetBuildConfig(Uri url)
     {
         var queries = HttpUtility.ParseQueryString(url.Query);
         var name = queries["name"] ?? "";
-        var repository = queries["repository_url"] ?? "";
-        var branch = queries["branch"] ?? "";
+        var repository = queries["publish_repository_url"] ?? "";
```

Comment:
>queries["publish_repository_url"] ?? "" [](http://example.com/codeflow?start=25&length=39)

We should fallback to "repository_url" if the `publish_repository_url` is null or empty, for the other docfx caller, like docs-ui, vscode extension, there is no `publish_repository_url`